package.json

```diff
@@ -46,8 +55,11 @@
     "build-common": "cd common && yarn build && cd ..",
     "build-maint": "cd maintenance && yarn build && cd ..",
     "import-locales": "node scripts/import-locales",
-    "lint": "prettier \"**/*.{ts,tsx}\"",
```

Comment:
ESLint now lints for Prettier so we don't need to do this check in the lint command anymore